File: 1036/1036.py
A, B, C = map(float, input().split())

bas = (B * B) - ( 4 * A * C)

#se bas for negativo, impossivel calcular
#se A !== 0, impossivel calcular
#else, faz o calculo

if (bas < 0 or A == 0):
    print("Impossivel calcular")
else:
    bas = bas ** (0.5)
    R1 = (-B + bas)/(2*A)
    R2 = (-B - bas)/(2*A)
    print(f"R1 = {R1:.5f}")
    print(f"R2 = {R2:.5f}")

File: 1036/1036.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var [A, B, C] = lines.shift().split(" ").map(Number);

var bas = B * B - 4 * A * C;

if (bas < 0 || A === 0) {
  console.log("Impossivel calcular");
} else {
  bas = Math.sqrt(bas);
  var R1 = (-B + bas) / (2 * A);
  var R2 = (-B - bas) / (2 * A);
  console.log("R1 = " + R1.toFixed(5));
  console.log("R2 = " + R2.toFixed(5));
}
